pkg/doc/signature/proof/proof.go

```diff
@@ -53,7 +53,7 @@ type Proof struct {
 func NewProof(emap map[string]interface{}) (*Proof, error) {
 	created := stringEntry(emap[jsonldCreated])
 
-	timeValue, err := time.Parse(time.RFC3339, created)
+	timeValue, err := time.Parse(time.RFC3339Nano, created)
```

Comment:
shouldn’t we also do this in the other time values in the VC data model? @sudeshrshetty @kdimak

Reply:
Yes, but this will be another PR - to close https://github.com/hyperledger/aries-framework-go/issues/1772

Reply:
We are using time.RFC3339 format for time everywhere.

Reply:
Why we need this one?